.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+# Purpose
+
+**Type of PR**
+* [ ] Bug Fix
+* [ ] Feature Work
+* [ ] Revert
+* [ ] Refactor / Code Cleanup
+* [ ] Other (fill in...)
+
+**Review Urgency**
+* [ ] ASAP
+* [ ] Next 2 hours
+* [ ] End of Day
+* [ ] Tomorrow
+* [ ] End of Week
+* [ ] Other (fill in...)
+
+
+## Short Description
+<!-- Feature / Problem overview -->
+
+
+## Changelog / Code Overview
+<!-- What was changed / added / removed and why. If you changed the frontend please include screenshots -->
+
+
+## Test Plan
+<!-- How was this code tested / How should reviewers test it? -->
+
+
+## Deploy Plan
```

Comment:
Probably not needed? Maybe remove for now and we can add it back in the future if we need to